google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -155,6 +156,32 @@ public Mutation setCell(
     return this;
   }
 
+  @Override
+  public Mutation setCell(@Nonnull String familyName, @Nonnull String qualifier, long value) {
+    return setCell(familyName, wrapByteString(qualifier), value);
+  }
+
+  @Override
+  public Mutation setCell(
+      @Nonnull String familyName, @Nonnull String qualifier, long timestamp, long value) {
+
+    return setCell(familyName, wrapByteString(qualifier), timestamp, value);
+  }
+
+  @Override
+  public Mutation setCell(@Nonnull String familyName, @Nonnull ByteString qualifier, long value) {
+    long timestamp = System.currentTimeMillis() * 1_000;
+
+    return setCell(familyName, qualifier, timestamp, value);
+  }
+
+  @Override
+  public Mutation setCell(
+      @Nonnull String familyName, @Nonnull ByteString qualifier, long timestamp, long value) {
+
+    return setCell(familyName, qualifier, timestamp, ByteString.copyFrom(Longs.toByteArray(value)));
```

Comment:
@igorbernstein2 I have relied upon `Longs.toByteArray(long value)` for long value conversion instead of ByteBuffer. As ByteBuffer needs to be `rewind()` before passing it to `ByteString.copyFrom(ByteBuffer byteBuffer)`.

I hope this is acceptable.

Reply:
Yep, thank you